### ISSUE TITLE
Please distribute tests in sdist tarball

### ISSUE THREAD

**Issue by scarabeusiv** [matze/pkgconfig#45]
We in distributions validate the packages against our stack of python and without the tests present we are not able to make sure that the pkgconfig module really works with what we distribute.
As you switched to poetry I can't just grab the github tag and expect the same results.

**Comment by matze** [comment]
I guess it's related to https://github.com/sdispater/poetry/issues/1109?

**Comment by scarabeusiv** [comment]
Yep, mostly. In setuptools you would tweak up manifest.in, but in poetry I dunno how to say it to distribute the stuff.

Note the tests should not be installed, just present in the archive for us tu use...

**Comment by arunpersaud** [comment]
+1
